q2_qemistree/tests/test_process_fingerprint.py

```diff
@@ -29,22 +30,30 @@ def setUp(self):
         self.goodcsi = qiime2.Artifact.load(os.path.join(THIS_DIR,
                                                          'data/csiFolder.qza'))
         properties_path = os.path.join(data, 'molecular_properties.csv')
-        self.properties = pd.read_table(properties_path, dtype=str)
+        self.properties = pd.read_csv(properties_path, dtype=str, sep='\t')
         self.properties.set_index('absoluteIndex', inplace=True)
 
     def test_fingerprintOut(self):
         msg = "Fingerprint file is empty!"
         with self.assertRaisesRegex(ValueError, msg):
             collate_fingerprint(self.emptycsi)
 
-    def test_featureMatch(self):
+    def test_featureMatch1(self):
```

Comment:
Could you rename these two test cases to be `test_feature_match_xxx` where xxxx is a short description of the test case variation?

Reply:
updated these tests